sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CodeGenerationSuite.scala

```diff
@@ -107,7 +107,7 @@ class CodeGenerationSuite extends SparkFunSuite with ExpressionEvalHelper {
       strExpr = StringDecode(Encode(strExpr, "utf-8"), "utf-8")
```

Comment:
can we use a different expression for testing? The codegen size is greatly decreased after using `StaticInvoke` in `Encode`.

Reply:
e.g. `StringTrim`

Reply:
Nice catch!